clients/browser/style.css

```diff
@@ -37,29 +48,102 @@ body {
 }
 
 header {
-    width: 250px;
+    width: 100%;
     max-width: 100%;
     margin: auto;
     margin-bottom: 25px;
     text-align: center;
 }
 
 header .name {
+      color:#FFF;
 	line-height: 85px;
```

Comment:
Since we're modifying this file anyway, it would be great if you could make the indentation consistent. :-)